docs/source/conf.py

```diff
@@ -3744,7 +3802,7 @@ def visit_reference(self, node):
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [(master_doc, "PyTorch", "PyTorch Documentation", [author], 1)]
+# man_pages = [(master_doc, "PyTorch", "PyTorch Documentation", [author], 1)]
```

Comment:
Why are you commenting out `man_pages` generation? (Though probably irrelevant)
If something does not work, would be nice to add TODO of sorts

Reply:
I brought it back. I don't think anyone actually uses it but I might be wrong.